website/package.json

```diff
@@ -15,18 +15,19 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.3",
-    "@docusaurus/preset-classic": "2.4.3",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.4.3",
-    "@mdx-js/react": "^1.6.22",
+    "@docusaurus/core": "3.2.0",
```

Comment:
I'm not sure if timing was just bad here, but when I run the doc site locally I'm getting warnings about it being out of date:
![image](https://github.com/STORIS/mvom/assets/4583869/a98cc800-a3ae-4a65-8ea8-82ed4a3150f5)

Reply:
Terrible timing haha Upgraded and updated the changelog.